frontend/src/app/uploads/DeviceLookup/DeviceLookup.tsx

```diff
@@ -0,0 +1,226 @@
+import { ComboBox } from "@trussworks/react-uswds";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faCheck, faCopy } from "@fortawesome/free-solid-svg-icons";
+
+import TextInput from "../../commonComponents/TextInput";
+import { DeviceType } from "../../../generated/graphql";
+import Optional from "../../commonComponents/Optional";
+import "./DeviceLookup.scss";
+
+interface Props {
+  formTitle: string;
+  deviceOptions?: DeviceType[];
+}
+
+const DeviceLookup = (props: Props) => {
+  const [device, updateDevice] = useState<DeviceType | undefined>();
+
+  const [copiedSlug, setCopiedSlug] = useState<string>();
+
+  useEffect(() => {
+    const timeout = copiedSlug
+      ? setTimeout(() => {
+          setCopiedSlug(undefined);
+        }, 3000)
+      : undefined;
+    return () => {
+      if (timeout) {
+        clearTimeout(timeout);
+      }
+    };
+  }, [copiedSlug]);
+
+  async function copySlug(slug: string) {
+    try {
+      await navigator.clipboard.writeText(slug);
+      setCopiedSlug(slug);
+    } catch (e: any) {
+      console.error(e);
+    }
+  }
+
+  const onChange = () => {};
+
+  const getDeviceOptions = () =>
+    props.deviceOptions
+      ? props.deviceOptions
+          .map((deviceType) => ({
+            label: deviceType.name,
+            value: deviceType.internalId,
+          }))
+          .sort((a, b) => a.label.localeCompare(b.label))
+      : [];
+
+  const getCopyToClipboardButton = (
+    copiedAttribute: string | undefined,
+    label: string
+  ) => {
+    return (
+      <div className="copy-button-container copy-button-tooltip">
+        {device && (
+          <button
+            className="usa-button usa-button--unstyled"
+            onClick={() => copiedAttribute && copySlug(copiedAttribute)}
+            aria-label={`${label}`}
+          >
+            <FontAwesomeIcon
+              icon={copiedSlug === copiedAttribute ? faCheck : faCopy}
+            />
+          </button>
+        )}
+        {device && copiedSlug === copiedAttribute && <CopyTooltip />}
+      </div>
+    );
+  };
+
+  return (
+    <div className="prime-home device-lookup-container flex-1">
+      <div className="grid-container">
+        <div className="grid-row">
+          <div className="prime-container card-container">
+            <div className="usa-card__header">
+              <div>
+                <h2>{props.formTitle}</h2>
+              </div>
+              <div
+                style={{
+                  display: "flex",
+                  justifyContent: "center",
+                  alignItems: "center",
+                }}
+              ></div>
+            </div>
+            <div className="usa-card__body margin-top-1">
+              {props.deviceOptions ? (
+                <div className="grid-row grid-gap">
+                  <div className="tablet:grid-col">
+                    <label htmlFor={"Select device"}>
+                      <ComboBox
```

Comment:
If the combox is inside of the label you can get rid of the `htmlFor` piece as that's supposed to tell what the label is describing. 
And you'll want to add back into the label `<LabeledText label={"Select device"} />` because right now there is no text above the input anymore
```suggestion
                    <label>
                       <LabeledText label={"Select device"} />
                      <ComboBox
```